api/src/certification/session-management/domain/models/AssessmentResultJuryComment.js

```diff
@@ -0,0 +1,17 @@
+import { AssessmentResult } from '../../../../shared/domain/models/AssessmentResult.js';
+
+export class AssessmentResultJuryComment {
+  /**
+   * @param {number} id
+   * @param {JuryComment} commentForCandidate
+   * @param {string} commentByJury
+   * @param {JuryComment} commentForOrganization
+   * @param {number} juryId
+   */
+  constructor({ id, commentByJury, juryId } = {}) {
+    this.id = id;
+    this.commentByJury = commentByJury;
+    this.juryId = juryId;
+    this.emitter = AssessmentResult.emitters.PIX_JURY;
+  }
+}
```

Comment:
La JSDoc n'est pas aligné avec les paramètres du constructeur de la classe

```suggestion
export class AssessmentResultJuryComment {
  /**
   * @param {number} id
   * @param {string} commentByJury
   * @param {number} juryId
   */
  constructor({ id, commentByJury, juryId } = {}) {
    this.id = id;
    this.commentByJury = commentByJury;
    this.juryId = juryId;
    this.emitter = AssessmentResult.emitters.PIX_JURY;
  }
}
```

Pas expert en JsDoc, je ne sais pas si on doit spécifier également `emitter` car il possède une valeur par défaut 🤷‍♂️

Reply:
Ce n'est pas un parametre et ici on ne documente que les parametres. On pourrait cependant aussi documenter les membres de la classe